tests/integration/targets/mso_version/tasks/connection.yml

```diff
@@ -0,0 +1,144 @@
+# Test code for the MSO modules
+# Copyright: (c) 2023, Akini Ross (@akinross) <akinross@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Set vars
```

Comment:
It would also be good to have a few tests related to adding and deleting objects to mso and check if the connection is maintained between them(without consecutive logins)

Reply:
What would be the benefit of this? The login flow would be the same, this is also why consecutive version queries are done and http_api logs are checked for the flow.